static/app/components/feedback/useFeedbackListQueryKey.tsx

```diff
@@ -33,18 +31,41 @@ export default function useFeedbackListQueryKey({organization}: Props): ApiQuery
     },
   });
 
-  const queryKey = useMemo(
-    (): ApiQueryKey => [
-      `/organizations/${organization.slug}/issues/`,
-      {
-        query: {
-          ...queryView,
-          query: `issue.category:feedback status:${queryView.mailbox} ${queryView.query}`,
-        },
-      },
-    ],
-    [organization, queryView]
-  );
+  const fixedQueryView = useMemo(() => {
+    // We don't want to use `statsPeriod` directly, because that will mean the
+    // start time of our infinite list will change, shifting the index/page
+    // where items appear if we invalidate the cache and refetch specific pages.
+    // So we'll convert statsPeriod into start/end time here, and use that. When
+    // the user wants to see fresher content (like, after the page has been open
+    // for a while) they can trigger that specifically.
+
+    const {statsPeriod, ...rest} = queryView;
+    const now = new Date();
+    const ms = intervalToMilliseconds(statsPeriod);
+    return statsPeriod
+      ? {
+          ...rest,
+          start: moment(now).subtract(ms, 'milliseconds').toISOString(),
```

Comment:
Is moment needed here?


```suggestion
          start: new Date(now - ms).toISOString(),
```
Or to be more explicit with `now`:

```suggestion
          start: new Date(now.getTime() - ms).toISOString(),
```

Reply:
d'oh! nice change